### PR TITLE
solution was added

### PR DESCRIPTION
**Pull request by Maksym845**
[DEMO LINK](https://Maksym845.github.io/js_get_data_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+    if (j % 3 === 0 && i !== 0) {
+      numbers[i] = ',' + numbers[i];
+    }
+  }
+
+  return numbers.join('');
+}
+
+const population = document.querySelectorAll('.population');
+const total = [...population].map(item => {
```
Comment (Denchepornyuk) [inline src/scripts/main.js:16]: you don't have to make a copy here, map is non-mutating method

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+  return Number(item.innerHTML.split(',').join(''));
+}).reduce((prev, item) => prev + item, 0);
+
+const average = total / [...population].length;
```
Comment (Denchepornyuk) [inline src/scripts/main.js:20]: the same here, you don't need array copy here

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]

File: src/scripts/main.js (hunk 1 above)
Comment (Maksym845) [reply src/scripts/main.js]: But here we get nodelist, with the spread operator I made the array from the nodelist.

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+  const numbers = String(number).split('');
+
+  for (let i = numbers.length - 1, j = 1; i >= 0; i--, j++) {
+    if (j % 3 === 0 && i !== 0) {
+      numbers[i] = ',' + numbers[i];
+    }
+  }
+
+  return numbers.join('');
+}
```
Comment (OleksiiBlezniuk) [inline src/scripts/main.js:13]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

**Review by OleksiiBlezniuk — APPROVED** [review]

**Pull request closed without merging**